2016-11-25-shiny-1.md

```diff
@@ -127,38 +127,38 @@ A basic `app.R` file consists of these parts:
   server <- function(input, output) {}
   ```
 
-- A command to run the app, this should be included at the end of every shiny app. It tells shiny that the user interface comes from the object called `ui` and that the server information (data, plots etc.) comes from the object called `server`.
+- A command to run the app, this should be included at the end of every shiny app. It tells Shiny that the user interface comes from the object called `ui` and that the server information (data, plots etc.) comes from the object called `server`.
 
   ```r
   shinyApp(ui = ui, server = server)
   ```
 
-Create a basic shiny app by copying the bits of code above into your `app.R`, delete any example code generated when you created the file.
+Create a basic shiny app by copying the bits of code above into your `app.R`, delete any example code generated when you created the file. 
 
 ## Layout of a Shiny App
 
 Shiny apps are structured using panels, which are laid out in different arrangements. Panels can contain text, widgets, plots, tables, images, etc.
 
 Below are some examples of panel layouts and the code that creates them:
 
-**_INSERT SLIDESHOW OF SHINY LAYOUTS_**
+**_INSERT SLIDESHOW OF SHINY LAYOUTS_** 
```

Comment:
Great idea to have the slideshow! Github flavoured Markdown slideshow tool - https://github.com/gnab/remark I haven't tried it out before, so not sure how easy it is to use, alternatively you could upload the photos on photobucket, make the slideshow there and then embed it.